### PR TITLE
(Feature) 468 server should send emails with new design

### PR DESCRIPTION
**Pull request by yaroslav-fedyshyn-nordwhale**
# Description

Implement design for recovery email, magic link and recovery instructions emails according to design.
Design: https://xd.adobe.com/spec/34f09e36-c7b1-48de-4506-1ef996f8fcd7-42c1/grid

Fixes https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/468

# How Has This Been Tested?

 - in GoodBootstrap/packages/server/src/server/verification/verificationAPI.js line 272 add development for onlyInEnv middleware.
- in GoodBootstrap/packages/server/src/server/send/sendAPI.js line 75 add development for onlyInEnv middleware.
- in GoodBootstrap/packages/server/src/server/send/sendAPI.js line 102 add development for onlyInEnv middleware.
- restart server
- then go and register a new wallet with some accessible email.
- you should receive 3 emails
- check the content/markup of it

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request ( for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

Screenshots:

----------------------------------------------------------------------------

----------------------------------------------------------------------------

----------------------------------------------------------------------------

----------------------------------------------------------------------------

----------------------------------------------------------------------------

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -32,3 +32,4 @@ FUSE_API=https://explorer.fuse.io
 TWILIO_AUTH_ID=
 TWILIO_AUTH_TOKEN=
 TWILIO_PHONE_NUMBER=
+DAPP_URL=http://localhost:3000
```
Comment (sirpy) [inline .env.dev]: this already exists as wallet_url

File: src/server/mautic/mauticAPI.js [hunk 2]
```diff
@@ -43,24 +43,38 @@ export const Mautic = {
     if (!(code && user.fullName && user.mauticId && Config.mauticVerifyEmailId))
       throw new Error('missing input for sending verification email')
 
+    const codeWithSpaces = code
+      .split('')
+      .filter(c => c)
+      .join('     ')
+
```
Comment (sirpy) [inline src/server/mautic/mauticAPI.js]: whats this?

File: src/server/verification/verificationAPI.js [hunk 3]
```diff
@@ -269,7 +269,7 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
   app.post(
     '/verify/sendemail',
     passport.authenticate('jwt', { session: false }),
-    onlyInEnv('production', 'staging', 'test'),
+    onlyInEnv('development', 'production', 'staging', 'test'),
```
Comment (sirpy) [inline src/server/verification/verificationAPI.js]: remove

File: src/server/mautic/mauticAPI.js (hunk 2 above)
Comment (yaroslav-fedyshyn-nordwhale) [reply src/server/mautic/mauticAPI.js]: removed it in a new commit. it was used for test

File: src/server/verification/verificationAPI.js (hunk 3 above)
Comment (yaroslav-fedyshyn-nordwhale) [reply src/server/verification/verificationAPI.js]: already removed

File: .env.dev (hunk 1 above)
Comment (yaroslav-fedyshyn-nordwhale) [reply .env.dev]: done

**Comment by sirpy** [comment]
the emails are not in the correct font

there's an extra line of "Please save it and keep it in a secure location." in the magiclink email
also in the tests please use 12 words seed phrase for the second+first parts

**Comment by yaroslav-fedyshyn-nordwhale** [comment]
> the emails are not in the correct font
> 
> there's an extra line of "Please save it and keep it in a secure location." in the magiclink email
> also in the tests please use 12 words seed phrase for the second+first parts

done

**Pull request merged**